scipy/sparse/_construct.py

```diff
@@ -301,16 +318,16 @@ def diags(diagonals, offsets=0, shape=None, format=None, dtype=None):
 def identity(n, dtype='d', format=None):
     """Identity matrix in sparse format
 
-    Returns an identity matrix with shape (n,n) using a given
+    Returns an identity matrix with shape ``(n, n)`` using a given
     sparse format and dtype. This differs from `eye_array` in
     that it has a square shape with ones only on the main diagonal.
-    It is thus the multiplicative identity. `eye_array` allows
+    It is thus the multiplicative identity. ``eye_array`` allows
     rectangular shapes and the diagonal can be offset from the main one.
 
     .. warning::
 
         This function returns a sparse matrix -- not a sparse array.
-        You are encouraged to use ``eye_array`` to take advantage
+        You are encouraged to use `eye_array` to take advantage
```

Comment:
Double backticks here?

Reply:
Single should be correct, since `eye_array` is a function. Something to check is whether the link to the function documentation renders.